web/app.js

```diff
@@ -465,6 +465,7 @@ const PDFViewerApplication = {
       pageColors,
       mlManager: this.mlManager,
       abortSignal: this._globalAbortController.signal,
+      enableHardwareAcceleration: AppOptions.get("enableHardwareAcceleration"),
```

Comment:
Shouldn't the thumbnail viewer also use this parameter?

Reply:
Good question. I wondered the same and I checked on my mac, it's almost 6 times slower to disable hwa.
For this specific case, I think it's fine to send the image in the gpu: it's a very basic operation which should work well.
So I think we shouldn't disable hwa here, I know it feels a bit inconsistent but it's likely better. That said it's something we change in the future if it's too bad.

Reply:
Sorry to ask, but did you actually test the "right" thing?
Please note that the thumbnail viewer may also *render* pages from scratch, and not just downsize existing ones from `PDFViewer`, and in that particular case I'd assume that this option would help just as in the viewer itself.

Reply:
Good point: I just tested resizing. I'll update my patch.